src/platform_impl/ios/app_state.rs

```diff
@@ -234,7 +234,7 @@ impl AppState {
         };
         self.set_state(AppStateImpl::ProcessingEvents {
             event_handler,
-            active_control_flow: ControlFlow::Poll,
+            active_control_flow: ControlFlow::default(),
```

Comment:
@madsmtm again not entirely sure what's going on here, if you don't mind having a second look I would appreciate that.

Reply:
Moved to https://github.com/rust-windowing/winit/pull/3107, as it seems that this is an issue unrelated to this PR.